reflex/components/radix/primitives/base.py

```diff
@@ -13,6 +13,9 @@ class RadixPrimitiveComponent(Component):
     # Change the default rendered element for the one passed as a child.
     as_child: Var[bool]
 
+    # "Fake" prop color_scheme is used to avoid shadowing CSS prop "color".
+    _rename_props: Dict[str, str] = {"colorScheme": "color"}
```

Comment:
i dont think we want this here, because no radix primitive will understand either `color_scheme` or `color` without our help.

i think having the `_excluded_props` is all we really need.

Reply:
Yeah that makes sense

Reply:
fixed